knative-operator/Gopkg.lock

```diff
@@ -363,6 +363,14 @@
   revision = "24b83195037b3bc61fcda2d28b7b0518bce293b6"
   version = "v1.0.4"
 
+[[projects]]
+  branch = "master"
+  digest = "1:0e9bfc47ab9941ecc3344e580baca5deb4091177e84dd9773b48b38ec26b93d5"
+  name = "github.com/mattbaird/jsonpatch"
```

Comment:
What is pulling this in? How many different jsonpatch libs do we need? :)

Reply:
Well, honestly, I don't really check what `dep` does in `.lock` file. I just added the upstream eventing operator as a dependency and jsonpatch lib came. 

I think this is something maybe we can address in the upstream operator but not here.